release_tester/test.py

```diff
@@ -53,6 +53,7 @@ def main(**kwargs):
                       kwargs['ssl']))
     print("V" * 80)
     status = True
+    test_driver.stop_monitor()
```

Comment:
I suggest to create a destructor for test_driver class and perform this call exactly in destruction phase. Currently, It is not DRY.

And we can forget about calling test_driver.stop_monitor() in case if we will add new test file

Reply:
sadly using destructors to do actual work is an anti-pattern in python :-(